src/date-range-picker/styles.scss

```diff
@@ -14,7 +14,12 @@ $calendar-header-color: awsui.$color-text-body-default;
 
 .root {
   @include styles.styles-reset;
-  max-width: 32em;
+  &:not(.wide) {
+    max-width: 32em;
```

Comment:
Scott' PR (https://github.com/cloudscape-design/components/pull/1851) for A-F components hasn't been merged yet, but I think we will need to update it to `max-inline-size`

Reply:
Thanks, updated.